pkg/v1/sdk/features/controllers/featuregate_controller.go

```diff
@@ -34,26 +34,26 @@ type FeatureGateReconciler struct {
 //+kubebuilder:rbac:groups=config.tanzu.vmware.com,resources=featuregates/status,verbs=get;update;patch
 
 // Reconcile reconciles the FeatureGate spec by computing activated, deactivated and unavailable features.
-func (r *FeatureGateReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), contextTimeout)
+func (r *FeatureGateReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	ctxCancel, cancel := context.WithTimeout(context.Background(), contextTimeout)
```

Comment:
You can use the passed in `ctx` to create a new context instead of `context.Background()`.